examples/opf/tools/sp_plotter.py

```diff
@@ -330,17 +330,16 @@ def testSPNew():
   
   
   # Setup a SP
-  sp = FDRCSpatial2.FDRCSpatial2(
```

Comment:
Did you run these examples before and after to make sure they still work as before?